go-controller/pkg/ovn/ovn.go

```diff
@@ -39,8 +39,9 @@ import (
 )
 
 const (
-	egressfirewallCRD    string = "egressfirewalls.k8s.ovn.org"
-	clusterPortGroupName string = "clusterPortGroup"
+	egressfirewallCRD             string = "egressfirewalls.k8s.ovn.org"
+	clusterPortGroupName          string = "clusterPortGroup"
+	clusterPortGroupMcastDenyName string = "mcastPortGroupDeny"
```

Comment:
nitpick: I think mcastDenyPortGroup sounds like a better name ?

Reply:
I guess but wouldn't that complicate code, during an upgrade wouldn't we also have to make sure to cleanup the port group that used the old name?

Reply:
@dceara Answering this question is bit tricky. For example for us, it should not be an issue because we didn't officially release the OVN yet (except one of the customer). Upstream code is all in master, so it depends on what commit is running in individuals cluster. @girishmg will this break your upgrade downstream?

I think it's probably an unnecessary risk at this point.